registry/storage/manifeststore.go

```diff
@@ -1,163 +0,0 @@
-package storage
-
-import (
-	"fmt"
-
-	"github.com/docker/distribution"
-	"github.com/docker/distribution/context"
-	"github.com/docker/distribution/digest"
-	"github.com/docker/distribution/manifest/schema1"
-	"github.com/docker/distribution/reference"
-	"github.com/docker/libtrust"
-)
-
-type manifestStore struct {
```

Comment:
See here for a preview: https://github.com/aaronlehmann/distribution/commit/cdfbed2784cb45434555e0a884752d2e6a54a2cc

Feel free to suggest a better name for `ManifestHandler`. Neither Richard nor I like it.